extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/common/MongoDatabaseResolver.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.mongodb.panache.common;
+
+/*
+ * This interface can be used to resolve the mongo database name in runtime, it'll help to implement multi-tenancy using tenant per database approach
```

Comment:
```suggestion
 * This interface can be used to resolve the MongoDB database name at runtime, it allows to implement multi-tenancy using a tenant per database approach.
```

Reply:
Thanks for your suggestion, I've added it and squashed all commits into one.